x/txreward/internal/types/Funder.go

```diff
@@ -0,0 +1,37 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+type Funder struct {
```

Comment:
Do we really need a `Funder` type when it can be represented by a simpler `sdk.AccAddress`? I really think we should delete this as it bring more complexity for no reason and makes the code more WET instead of [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)